src/components/10-atoms/carousel/child.scss

```diff
@@ -1,16 +1,16 @@
 @mixin defaultAttributesChildElements {
-  @include typo(primary);
+  @include text(t4);
   display: flex;
   flex-direction: column;
   text-align: center;
   justify-content: center; // align the dom element itself (f.e. small spans)
 
-  @include breakpoint($mediaquery-xs-up) {
-    @include typo(primary, large);
+  @include breakpoint($mediaquery-sm-up) {
```

Comment:
You changed the behaviour from xs to sm. Because the breakpoint also changed or a mistake?

Reply:
done